app/src/main.rs

```diff
@@ -19,42 +19,14 @@ defmt::timestamp! {"{=u64}", {
 
 #[entry]
 fn main() -> ! {
-    info!("Hello asd!");
-
+    info!("Hello from app!");
     let p = pac::Peripherals::take().unwrap();
 
     p.CACHE_S.enable.write(|w| w.enable().enabled());
 
     p.CLOCK_S.hfclkctrl.write(|w| w.hclk().div1());
     //p.CLOCK_S.hfclksrc.write(|w| w.src().hfxo());
 
-    if !p.UICR_S.approtect.read().pall().is_unprotected() {
```

Comment:
why remove these? (same for net)

Reply:
Oops, I was testing if the probe-rs automatically did that because It was never running on my board.
But I couldn't find any evidence on the source code, so I'm putting it back